packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -448,6 +461,8 @@ export function getValidatorApi({
         slot,
         isBuilderEnabled,
         strictFeeRecipientCheck,
+        // winston logger doesn't like bigint
+        builderBoostFactor: `${builderBoostFactor}`,
```

Comment:
`builderBoostFactor .toString()` might be more straightforward?

Reply:
hmmm could be, can add to todo cleanups